feature-detects/indexeddb2.js

```diff
@@ -0,0 +1,22 @@
+/*!
+{
+  "name": "IndexedDB 2.0",
+  "property": "indexeddb2",
+  "tags": ["storage"],
+  "caniuse": "indexeddb2",
+  "authors": ["Tan Zhen Yong (@Xenonym)"],
+  "polyfills": ["indexeddb"],
```

Comment:
is that polyfill also working for indexedDB 2.0?

Reply:
Yep! IndexedDBShim has 2.0 only methods like [`getAll()` and `getAllKeys()`](https://github.com/indexeddbshim/IndexedDBShim/blob/master/src/IDBIndex.js#L429).